### PR TITLE
Add macOS Admin Tools

### PR DESCRIPTION
**Pull request by smashism**
**https://github.com/smashism/awesome-macadmin-tools**

**This list is the best of the best tools available, free, paid, or open source, for people who admin Macs at organizations.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of sub-lists, preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by koppor** [comment]
Please go through the checklist.

- [ ] The repo should have awesome-list & awesome as GitHub topics. I encourage you to add more relevant topics.

The format with images is uncommon. Proposal create a "usual" awesomelist in `readme.md` and an extended one with images in `readme-extended.md`, where you link at the bottom of the list.

This entry of your list doesn't seem to be awesome, is it?

grafik

**Comment by smashism** [comment]
RE pictures, I based my list off of the Awesome macOS Screensavers list, which also has large images. I felt the illustration of the tools helps show how awesome they are, which is why they are included. I will review the other items. Thanks for the feedback.

**Comment by koppor** [comment]
Excerpt from the [curated list of screensavers for Mac OS X](https://github.com/agarrharr/awesome-macos-screensavers)

grafik

Title, Description, linked Image

Your list:

grafik

I would remove cost and add the link to the image/animated gif. So you have the same format  and the list would appear very lean.

**Comment by smashism** [comment]
I actually had it that way initially, but got feedback from users of the list that it wasn't inherent that clicking the images triggered a link. Users also asked for cost to be more clearly established in the list. They were added largely for accessibility reasons, especially for those that are visually impaired and use screen readers and other assistance tools.

If adding items for accessibility reasons doesn't make this list eligible to be "awesome" I'll just leave it unlisted and continue to share within the macadmin community via social media and other means.

**Comment by koppor** [comment]
I'd propose to fix the other things and then ask the maintainer of this
list.

Refs
https://github.com/sindresorhus/awesome/issues/1323#issuecomment-394661729

emily <notifications@github.com> schrieb am Do., 21. Juni 2018, 19:53:

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/smashism/awesome-macadmin-tools
✖ Linting

  readme.md:153:62
  ✖    1:28  Invalid badge source                                                         remark-lint:awesome/badge
  ✖   11:1   ToC item "Text Editing" does not match corresponding heading "Helper Menus"  remark-lint:awesome/toc
  ✖   47:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖   57:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖   67:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖   77:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖   97:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  107:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  121:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  153:62  Link to https://oandg.onfastspring.com/nomad-pro is dead                     remark-lint:no-dead-urls
  ✖  155:11  Link to https://nomad.menu/nomad-open-source/ is dead                        remark-lint:no-dead-urls
  ✖  157:1   Link to https://nomad.menu/nomad-open-source/ is dead                        remark-lint:no-dead-urls
  ✖  165:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  175:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  185:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  263:12  Text "osx" should be written as "macOS"                                      remark-lint:awesome/spell-check
  ✖  327:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  337:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  347:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  357:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  367:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  397:12  Text "github" should be written as "GitHub"                                  remark-lint:awesome/spell-check
  ✖  406:2   License image must be SVG                                                    remark-lint:awesome/license

  23 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: readme.md [hunk 1]
```diff
@@ -84,6 +84,7 @@
 - [macOS](https://github.com/iCHAIT/awesome-macOS)
 	- [Command-Line](https://github.com/herrbischoff/awesome-macos-command-line)
 	- [Screensavers](https://github.com/agarrharr/awesome-macos-screensavers)
+	- [Macadmin Tools](https://github.com/smashism/awesome-macadmin-tools)
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
	- [Admin Tools](https://github.com/smashism/awesome-macadmin-tools#readme)
```

**Comment by sindresorhus** [comment]
You should rename the repo from `https://github.com/smashism/awesome-macadmin-tools` to `https://github.com/smashism/awesome-mac-admin-tools` for readability.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**